javascript/ql/src/Security/CWE-078/UselessUseOfCat.ql

```diff
@@ -0,0 +1,25 @@
+/**
+ * @name Useless use of cat
```

Comment:
`Useless` is used a fair bit in query names, but `Useless use` is not used anywhere else. 

How about `Unnecessary use of cat`?

Reply:
There's [`The Useless Use of Cat Award`](http://porkmail.org/era/unix/award.html#cat) which inspired this query, so it would be nice to keep some semblance.

Perhaps we should have a more ordinary name for the query, and then add `("Useless Use of Cat")` like we did for `js/zip-slip`.

```
* @name Arbitrary file write during zip extraction ("Zip Slip")
```

Reply:
(I wrote that reply before seeing Erik's reply)

Reply:
I leave the resolution of this to the two of you, I said what I had to say from an editorial point of view and as a non-developer) 😃

Reply:
@esbena how do you feel about `Unnecessary use of cat`?

Reply:
Lets qualify it a bit more: 
```
Unnecessary use of `cat` process
```